source/common/config/filter_json.cc

```diff
@@ -171,11 +171,9 @@ void FilterJson::translateHttpConnectionManager(
     const auto json_tracing = json_config.getObject("tracing");
     auto* tracing = proto_config.mutable_tracing();
 
-    envoy::config::filter::network::http_connection_manager::v2::HttpConnectionManager::Tracing::
-        OperationName operation_name{};
-    envoy::config::filter::network::http_connection_manager::v2::HttpConnectionManager::Tracing::
-        OperationName_Parse(StringUtil::toUpper(json_tracing->getString("operation_name")),
-                            &operation_name);
+    envoy::api::v2::core::TracingOperation operation_name{};
```

Comment:
I don't think we need this change since this is for the deprecated V1 config handling

Reply:
I renamed the proto enum, so this is necessary for build to complete. It has no effect on the wire, since it's the same enum structurally.

Reply:
ah yeah that makes sense